### DISCUSSION TITLE
New Backend Developer Guide.

### DISCUSSION THREAD

**Discussion by oyekamal** [Banbury-inc/Athena#66] [General]
Hi, I'm backed developer i would like to working on the project. but i don't know where to start and what to implement?

**Comment by oyekamal** [comment]
@mmills6060 if you can guide me i can write the complete docker configuration for FE, BE and also the documentation too, so anyone can start working on the project easily.

**Reply by mmills6060** [reply]
Thanks for the help! Information about what we are working on can be found here: https://github.com/orgs/Banbury-inc/projects/2

It sounds like you already have an idea of what to work on. If you create a pull request, I would be happy to review and approve. I usually get to the requests in a few hours. We are very close to finishing the prototype!
